src/Components/NearBy/Component/Dimmer.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import "./Dimmer.scss";
+
+class Dimmer extends Component {
+  render() {
+    const { modalOff } = this.props;
+    return;
+    <div className="Dimmer" onClick={modalOff}></div>;
```

Comment:
```suggestion
    <div className="Dimmer" onClick={modalOff} />
```
- 이런 기본적인 것들 검토하는 습관 기르시면 코드도 깔끔해지고 안 풀리던 문제도 해결되고 그렇습니다-!

Reply:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋ
네 알겠습니다... 셀프클로징!